### PR TITLE
Add util.egg

### PR DESCRIPTION
**Pull request by clyben**
Add util.egg, which contains ListExpr-length and ListExpr-ith function.

### CODE REVIEW

File: tree_unique_args/src/util.egg [hunk 1]
```diff
@@ -0,0 +1,13 @@
+(function ListExpr-length (ListExpr) i64)
+(function ListExpr-ith (ListExpr i64) Expr :unextractable)
+(function ListExpr-surfix (ListExpr i64) ListExpr :unextractable)
```
Comment (oflatt) [inline tree_unique_args/src/util.egg:3]: Typo: I think you meant "suffix"

**Review by oflatt — CHANGES_REQUESTED** [review]
Looks good! I think you have a typo

File: tree_unique_args/src/util.egg (hunk 1 above)
Comment (clyben) [reply tree_unique_args/src/util.egg:3]: ohh....(

**Review by oflatt — APPROVED** [review]
Thanks!
For future reference, after you fix up a PR don't forget to re-request a review.

**Pull request merged**